data_portal/models.py

```diff
@@ -393,3 +393,66 @@ class Meta:
 
     def __str__(self):
         return f"WORKFLOW_RUN_ID: {self.wfr_id}, WORKFLOW_TYPE: {self.type_name}, WORKFLOW_START: {self.start}"
+
+
+class Report(models.Model):
+    report_id = models.IntegerField(auto_created=True);
+    sample_id = models.ForeignKey(LIMSRow, on_delete=models.CASCADE, null=True, blank=True)
```

Comment:
Ah wait! this should also be `limsrow` or `lims_row` instead of `sample_id` as this is in an object reference (i.e. ORM keep forgetting about this). So it has to instantiate `LIMSRow` object (get it from db) first before creating a Report. e.g. Pseudocode:
```
parse_and_extract_sample_id_subject_id_from_file_name()
xy_lims_row = LIMSRow.get(subject_id='X', sample_id='Y')
report = Report(lims_rows=xy_lims_row, ...)
```

Reply:
We also need to consider what make this Report model unique? Might need some advice from domain expert! ;) It could be a combination of couple of fields i.e. class Meta `unique_together = ['field1', 'field2', 'field3']`. Otherwise, we may potentially be inserting duplicate records...